user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr :hoge
+  def initialize
+    @hoge =    [a,b,c,d]*''
+    # にゃーん
+  end
+
+  def get_hoge
+    @hoge
```

Comment:
__[staging][rubocop]__
* Do not prefix reader method names with `get_`. [:ref](https://github.com/bbatsov/ruby-style-guide#accessor_mutator_method_names)

<!-- Comment by [SideCI](https://sideci.com) -->